tests/lib/tools/suite/fs-state/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: smoke test for the fs test tool
+execute: |
+    # Check help
+    "$TESTSTOOLS"/fs-state -h | MATCH "usage: fs-state wait-for-file"
```

Comment:
Please test the version without arguments as well.

Reply:
done